.changeset/four-kids-nail.md

```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/polaris': minor
+---
+
+Enable overlays to work inside React portals
```

Comment:
Can we write this in passed tense? Also I'd prefer if we were a bit more specific in what this fixed